musicdroid/src/main/java/org/catrobat/musicdroid/pocketmusic/projectselection/ProjectSelectionActivity.java

```diff
@@ -108,9 +114,36 @@ public boolean onOptionsItemSelected(MenuItem item) {
             return true;
         }
 
+        if(id == R.id.action_about) {
+            displayDialog();
+        }
         return super.onOptionsItemSelected(item);
     }
 
+    private void displayDialog() {
+        editTextProjectName = new EditText(this);
+        AlertDialog.Builder alertDialogBuilder = new AlertDialog.Builder(this);
```

Comment:
Please do not use inline dialogs. The location for dialogs is: projectselection/dialog